BE/Startup.cs

```diff
@@ -1,67 +1,91 @@
-using System.Linq;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.EntityFrameworkCore;
-using System.Text;
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.OpenApi.Models;
+using BE.Queries.Revisions.GetAllRevisions;
+using BE.Queries.Revisions.GetRevisionById;
+using BE.BL.Revisions.Create;
+using BE.BL.Revisions.Edit;
+using BE.Queries.Cars;
+using BE.BL.Cars.Create;
+using BE.BL.Cars.Edit;
+using BE.Queries.Revisions.GetAllRevisionsByClientId;
 
 namespace BE
 {
     public class MyContext : DbContext
```

Comment:
Rename to something more interesting, maybe SqlDbContext or GetYourCarFixedDbContext